### ISSUE TITLE
Color is off

### ISSUE THREAD

**Issue by pipe01** [sskodje/ScreenRecorderLib#27]
It seems like the recording's color is much more whiter than the screen actually is. This doesn't happen with any recording software. Here are a couple shots demonstrating the issue:

Original:
![Captura de pantalla (24)](https://user-images.githubusercontent.com/3885705/54452060-7c50b600-4754-11e9-909b-65a6862fe64c.png)

Recording:
![vlcsnap-2019-03-15-18h52m28s680](https://user-images.githubusercontent.com/3885705/54452074-896da500-4754-11e9-95fb-947bf58d8ed5.png)

These are my options:
```cs
Rec = Recorder.CreateRecorder(new RecorderOptions
{
    VideoOptions = new VideoOptions
    {
        BitrateMode = BitrateControlMode.Quality,
        Bitrate = 8000 * 1000,
        Framerate = 60,
        Quality = 70,
        IsMousePointerEnabled = true,
        IsFixedFramerate = true,
        EncoderProfile = H264Profile.Main
    },
    AudioOptions = new AudioOptions
    {
        Bitrate = AudioBitrate.bitrate_128kbps,
        Channels = AudioChannels.Stereo,
        IsAudioEnabled = true
    },
    IsLowLatencyEnabled = true,
    IsFragmentedMp4Enabled = true,
    IsMp4FastStartEnabled = true,
    RecorderMode = RecorderMode.Video
});
```

The issue persists when disabling `IsLowLatencyEnabled`, `IsFragmentedMp4Enabled` and `IsMp4FastStartEnabled`. Thanks!

**Comment by sskodje** [comment]
Hello! I believe this is a color space issue. I had some issues with hardware encoding and this issue, but i thought it was fixed when i changed video format to YUY2 a couple versions ago. Do the problem persist if you disable hardware encoding, and what is the graphics card that you encode with?

**Comment by pipe01** [comment]
I'm using a 2400G with its Vega 11 as the GPU. This what I get when I turn hardware acceleration off:
![vlcsnap-2019-03-16-00h50m05s403](https://user-images.githubusercontent.com/3885705/54467466-8c34be00-4785-11e9-8483-65dbc3f0024a.png)
I think I accidentally entered a creepypasta

**Comment by sskodje** [comment]
I've actually had a red screen of doom like that before, but it got fixed with a reboot. I have no idea what caused it. Software encoding should work on all systems..

**Comment by sskodje** [comment]
Well it seems to be an issue with color ranges in the AMD encoder. I'm still working on it, but i uploaded a fix for the broken software encoder, so you can use that in the mean time if possible.

**Comment by pipe01** [comment]
I've been using FFmpeg for the time being, when/if you're able to fix it I'll probably use both.

**Comment by sskodje** [comment]
This issue should be fixed with 9aeb3349e1403e5a7cc09b128f19852f9a3e5cc8.
